core/controllers/classifier.py

```diff
@@ -122,7 +122,8 @@ def post(self):
         # signature consistent on both Oppia and Oppia-ml.
         # For more info visit: https://stackoverflow.com/q/40173295
         classifier_data = (
-            classifier_services.convert_strings_to_float_numbers_in_classifier_data( #pylint: disable=line-too-long
+            classifier_services.
+            convert_strings_to_float_numbers_in_classifier_data(
```

Comment:
Is this OK to break after the dot? (Here and everywhere)

Reply:
If we don't use these then we have to use pylint pragmas, so instead of using pylint pragma, I think it is better to do this.

Reply:
Yes it is fine to break after the dot.